test/unit/messengers/cinstance_messenger_test.rb

```diff
@@ -183,7 +183,8 @@ def test_plan_change
       assert messenger.deliver
       assert provider.received_messages.empty?
 
-      message = master.received_messages.last!
+      # Using first! as they are ordered by created_at desc
```

Comment:
This table does not have a `created_at` so this comment kills me :joy: 
https://github.com/3scale/porta/blob/e2dbb0cf87ed7a340261f681b9444254dd1a199b/db/schema.rb#L799-L809